app/views/play/cinematic/Command/AbstractCommand.js

```diff
@@ -0,0 +1,91 @@
+import * as Promise from 'bluebird'
```

Comment:
Is `Promise` not available globally?

Reply:
Bluebird Promises are different and include cancellation. Native ones don't.